src/test/java/uk/gov/hmcts/reform/civil/handler/callback/user/HearingScheduledHandlerTest.java

```diff
@@ -164,7 +164,9 @@ void shouldGetDueDateAndFeeSmallClaim_whenAboutToSubmit() {
             var response = (AboutToStartOrSubmitCallbackResponse) handler.handle(params);
             CaseData updatedData = mapper.convertValue(response.getData(), CaseData.class);
             assertThat(updatedData.getHearingFee()).isEqualTo(
+
```

Comment:
Only white-space changes here. If you rollback this file, it's one less to be reviewed.